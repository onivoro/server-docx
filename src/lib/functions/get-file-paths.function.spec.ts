import { getFilePaths } from './get-file-paths.function';

describe('getFilePaths', () => {
    describe('creates a set of unique but conventional strings for inflating/deflating docx files on disk', () => {

        it('without a prefix', () => {
            expect(getFilePaths('asdf.docx')).toMatchSnapshot();
        });

        it('with a prefix', () => {
            expect(getFilePaths('asdf.docx', 'some-dir')).toMatchSnapshot();
        });
    });
});