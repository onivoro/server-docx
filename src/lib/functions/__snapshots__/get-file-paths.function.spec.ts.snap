// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getFilePaths creates a set of unique but conventional strings for inflating/deflating docx files on disk with a prefix 1`] = `
{
  "contentPath": "some-dir/asdf/inflated/word/document.xml",
  "inflated": "some-dir/asdf/inflated",
  "inputFilePath": "some-dir/asdf/asdf.docx",
  "outputFileName": "some-dir/asdf-modified.docx",
  "outputFilePath": "some-dir/asdf/asdf-modified.docx",
  "root": "some-dir/asdf",
}
`;

exports[`getFilePaths creates a set of unique but conventional strings for inflating/deflating docx files on disk without a prefix 1`] = `
{
  "contentPath": "asdf/inflated/word/document.xml",
  "inflated": "asdf/inflated",
  "inputFilePath": "asdf/asdf.docx",
  "outputFileName": "asdf-modified.docx",
  "outputFilePath": "asdf/asdf-modified.docx",
  "root": "asdf",
}
`;
